Docs/SampleCustom.plist

```diff
@@ -1001,6 +1001,8 @@
 			<false/>
 			<key>AppleXcpmForceBoost</key>
 			<false/>
+			<key>CustomPCISerialDevice</key>
```

Comment:
Pci to match others.

Reply:
https://github.com/acidanthera/OpenCorePkg/commit/321a06d855987a17d4d23025cfa562011ece1599

Reply:
https://github.com/acidanthera/OpenCorePkg/commit/321a06d855987a17d4d23025cfa562011ece1599